## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 454616`
Commit: `1977293be3f9d4b1bd83066c8b3b14a642f02e32`

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>jellyfin-media-player</li>
  </ul>
</details>
<details>
  <summary>:x: 8 packages failed to build:</summary>
  <ul>
    <li>gpodder</li>
    <li>persepolis</li>
    <li>pinchflat</li>
    <li>previewqt</li>
    <li>switchfin</li>
    <li>ytcc</li>
    <li>ytcc.dist</li>
    <li>ytui-music</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 285 packages built:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>ani-cli</li>
    <li>auto-editor</li>
    <li>backgroundremover</li>
    <li>backgroundremover.dist</li>
    <li>beanprice</li>
    <li>beanprice.dist</li>
    <li>botamusique</li>
    <li>catt</li>
    <li>catt.dist</li>
    <li>cgt-calc</li>
    <li>cgt-calc.dist</li>
    <li>cplay-ng</li>
    <li>cplay-ng.dist</li>
    <li>curseradio</li>
    <li>curseradio.dist</li>
    <li>dmlive</li>
    <li>dra-cla</li>
    <li>ff2mpv</li>
    <li>ff2mpv-go</li>
    <li>gallery-dl</li>
    <li>gallery-dl.dist</li>
    <li>gonic</li>
    <li>gophertube</li>
    <li>gtk-pipe-viewer</li>
    <li>gtk-pipe-viewer.devdoc</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>invidtui</li>
    <li>jellyfin-tui</li>
    <li>jellytui</li>
    <li>kodiPackages.sendtokodi</li>
    <li>moodle-dl</li>
    <li>moodle-dl.dist</li>
    <li>mopidy-youtube (mopidyPackages.mopidy-youtube)</li>
    <li>mopidy-youtube.dist (mopidyPackages.mopidy-youtube.dist)</li>
    <li>mov-cli</li>
    <li>mov-cli.dist</li>
    <li>mpv</li>
    <li>mpvScripts.mpv-playlistmanager</li>
    <li>parrot</li>
    <li>pipe-viewer</li>
    <li>pipe-viewer.devdoc</li>
    <li>python312Packages.baselines</li>
    <li>python312Packages.baselines.dist</li>
    <li>python312Packages.bgutil-ytdlp-pot-provider</li>
    <li>python312Packages.bgutil-ytdlp-pot-provider.dist</li>
    <li>python312Packages.bsuite</li>
    <li>python312Packages.bsuite.dist</li>
    <li>python312Packages.curl-cffi</li>
    <li>python312Packages.curl-cffi.dist</li>
    <li>python312Packages.fastai</li>
    <li>python312Packages.fastai.dist</li>
    <li>python312Packages.gym</li>
    <li>python312Packages.gym.dist</li>
    <li>python312Packages.k-diffusion</li>
    <li>python312Packages.k-diffusion.dist</li>
    <li>python312Packages.llama-cloud-services</li>
    <li>python312Packages.llama-cloud-services.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-core</li>
    <li>python312Packages.llama-index-core.dist</li>
    <li>python312Packages.llama-index-embeddings-gemini</li>
    <li>python312Packages.llama-index-embeddings-gemini.dist</li>
    <li>python312Packages.llama-index-embeddings-google</li>
    <li>python312Packages.llama-index-embeddings-google.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llama-index-embeddings-ollama</li>
    <li>python312Packages.llama-index-embeddings-ollama.dist</li>
    <li>python312Packages.llama-index-embeddings-openai</li>
    <li>python312Packages.llama-index-embeddings-openai.dist</li>
    <li>python312Packages.llama-index-graph-stores-nebula</li>
    <li>python312Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python312Packages.llama-index-graph-stores-neo4j</li>
    <li>python312Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python312Packages.llama-index-graph-stores-neptune</li>
    <li>python312Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python312Packages.llama-index-legacy</li>
    <li>python312Packages.llama-index-legacy.dist</li>
    <li>python312Packages.llama-index-llms-ollama</li>
    <li>python312Packages.llama-index-llms-ollama.dist</li>
    <li>python312Packages.llama-index-llms-openai</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-llms-openai.dist</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python312Packages.llama-index-node-parser-docling</li>
    <li>python312Packages.llama-index-node-parser-docling.dist</li>
    <li>python312Packages.llama-index-readers-database</li>
    <li>python312Packages.llama-index-readers-database.dist</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-json</li>
    <li>python312Packages.llama-index-readers-json.dist</li>
    <li>python312Packages.llama-index-readers-llama-parse</li>
    <li>python312Packages.llama-index-readers-llama-parse.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index-readers-twitter</li>
    <li>python312Packages.llama-index-readers-twitter.dist</li>
    <li>python312Packages.llama-index-readers-txtai</li>
    <li>python312Packages.llama-index-readers-txtai.dist</li>
    <li>python312Packages.llama-index-readers-weather</li>
    <li>python312Packages.llama-index-readers-weather.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index-vector-stores-google</li>
    <li>python312Packages.llama-index-vector-stores-google.dist</li>
    <li>python312Packages.llama-index-vector-stores-milvus</li>
    <li>python312Packages.llama-index-vector-stores-milvus.dist</li>
    <li>python312Packages.llama-index-vector-stores-qdrant</li>
    <li>python312Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llama-parse</li>
    <li>python312Packages.llama-parse.dist</li>
    <li>python312Packages.moviepy</li>
    <li>python312Packages.moviepy.dist</li>
    <li>python312Packages.mpv</li>
    <li>python312Packages.mpv.dist</li>
    <li>python312Packages.skrl</li>
    <li>python312Packages.skrl.dist</li>
    <li>python312Packages.spacy</li>
    <li>python312Packages.spacy-curated-transformers</li>
    <li>python312Packages.spacy-curated-transformers.dist</li>
    <li>python312Packages.spacy-loggers</li>
    <li>python312Packages.spacy-loggers.dist</li>
    <li>python312Packages.spacy-lookups-data</li>
    <li>python312Packages.spacy-lookups-data.dist</li>
    <li>python312Packages.spacy-transformers</li>
    <li>python312Packages.spacy-transformers.dist</li>
    <li>python312Packages.spacy.dist</li>
    <li>python312Packages.textacy</li>
    <li>python312Packages.textacy.dist</li>
    <li>python312Packages.textnets</li>
    <li>python312Packages.textnets.dist</li>
    <li>python312Packages.tubeup</li>
    <li>python312Packages.tubeup.dist</li>
    <li>python312Packages.wandb</li>
    <li>python312Packages.wandb.dist</li>
    <li>python312Packages.yark</li>
    <li>python312Packages.yark.dist</li>
    <li>python312Packages.yfinance</li>
    <li>python312Packages.yfinance.dist</li>
    <li>python312Packages.yt-dlp</li>
    <li>python312Packages.yt-dlp-light</li>
    <li>python312Packages.yt-dlp-light.dist</li>
    <li>python312Packages.yt-dlp.dist</li>
    <li>python313Packages.baselines</li>
    <li>python313Packages.baselines.dist</li>
    <li>python313Packages.bgutil-ytdlp-pot-provider</li>
    <li>python313Packages.bgutil-ytdlp-pot-provider.dist</li>
    <li>python313Packages.bsuite</li>
    <li>python313Packages.bsuite.dist</li>
    <li>python313Packages.curl-cffi</li>
    <li>python313Packages.curl-cffi.dist</li>
    <li>python313Packages.fastai</li>
    <li>python313Packages.fastai.dist</li>
    <li>python313Packages.gym</li>
    <li>python313Packages.gym.dist</li>
    <li>python313Packages.k-diffusion</li>
    <li>python313Packages.k-diffusion.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-node-parser-docling</li>
    <li>python313Packages.llama-index-node-parser-docling.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-milvus</li>
    <li>python313Packages.llama-index-vector-stores-milvus.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.moviepy</li>
    <li>python313Packages.moviepy.dist</li>
    <li>python313Packages.mpv</li>
    <li>python313Packages.mpv.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.spacy</li>
    <li>python313Packages.spacy-curated-transformers</li>
    <li>python313Packages.spacy-curated-transformers.dist</li>
    <li>python313Packages.spacy-loggers</li>
    <li>python313Packages.spacy-loggers.dist</li>
    <li>python313Packages.spacy-lookups-data</li>
    <li>python313Packages.spacy-lookups-data.dist</li>
    <li>python313Packages.spacy-transformers</li>
    <li>python313Packages.spacy-transformers.dist</li>
    <li>python313Packages.spacy.dist</li>
    <li>python313Packages.textacy</li>
    <li>python313Packages.textacy.dist</li>
    <li>python313Packages.tubeup</li>
    <li>python313Packages.tubeup.dist</li>
    <li>python313Packages.wandb</li>
    <li>python313Packages.wandb.dist</li>
    <li>python313Packages.yark</li>
    <li>python313Packages.yark.dist</li>
    <li>python313Packages.yfinance</li>
    <li>python313Packages.yfinance.dist</li>
    <li>yt-dlp (python313Packages.yt-dlp)</li>
    <li>yt-dlp-light (python313Packages.yt-dlp-light)</li>
    <li>yt-dlp-light.dist (python313Packages.yt-dlp-light.dist)</li>
    <li>yt-dlp.dist (python313Packages.yt-dlp.dist)</li>
    <li>radio-cli</li>
    <li>shira</li>
    <li>shira.dist</li>
    <li>somafm-cli</li>
    <li>spotdl</li>
    <li>spotdl.dist</li>
    <li>subtitleedit</li>
    <li>szurubooru.server</li>
    <li>szurubooru.server.dist</li>
    <li>tomato-c</li>
    <li>tts</li>
    <li>tts.dist</li>
    <li>video-downloader</li>
    <li>wtwitch</li>
    <li>yewtube</li>
    <li>yewtube.dist</li>
    <li>youtube-tui</li>
    <li>ytdl-sub</li>
    <li>ytdl-sub.dist</li>
    <li>ytdownloader</li>
    <li>ytfzf</li>
    <li>ytmdl</li>
    <li>ytmdl.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>gpodder</summary>
<pre>    
                    if sys.platform == &quot;darwin&quot; and port == 5000:
                        print(
                            &quot;On macOS, try disabling the &#x27;AirPlay Receiver&#x27; service&quot;
                            &quot; from System Preferences -&gt; General -&gt; AirDrop &amp; Handoff.&quot;,
                            file=sys.stderr,
                        )
    
&gt;               sys.exit(1)
E               SystemExit: 1

/nix/store/piaj774py4hss30i9kc0cqds2ixnbwx5-python3.13-werkzeug-3.1.3/lib/python3.13/site-packages/werkzeug/serving.py:782: SystemExit
---------------------------- Captured stderr setup -----------------------------
Operation not permitted
=========================== short test summary info ============================
ERROR tests/test_feedcore.py::test_easy - SystemExit: 1
ERROR tests/test_feedcore.py::test_redirect - SystemExit: 1
ERROR tests/test_feedcore.py::test_redirect_loop - SystemExit: 1
ERROR tests/test_feedcore.py::test_temporary_error_retry - SystemExit: 1
============================== 4 errors in 0.58s ===============================</pre>
</details>
<details>
<summary>pinchflat</summary>
<pre>Running phase: unpackPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;unpackPhase&quot; }
unpacking source archive /nix/store/j30nq31xbhjcaychg44ygk6w0igfwr37-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
** (Mix) failed to open a TCP socket while acquiring a lock, reason: :eperm</pre>
</details>
<details>
<summary>previewqt</summary>
<pre>-- Found OpenGL: /nix/store/ybygcq3s191f07bwgxbqpgqv0ak4h17k-apple-sdk-11.3/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/OpenGL.framework
-- Found WrapOpenGL: TRUE
-- Found WrapVulkanHeaders: /nix/store/7nm3m6bfxij8zyy4rp9jf2ixxxdr76nh-vulkan-headers-1.4.328.0/include
-- Found Cups: /nix/store/sql25h6l2vvh4vyin4k38ncnmfil202a-cups-2.4.14-lib/lib/libcups.dylib (found version &quot;2.4.14&quot;)
-- Found ImageMagick: /nix/store/qzba5k943034hbdqqqzjsd0g2bd3zflj-imagemagick-7.1.2-8/lib/libMagick++-7.Q16HDRI.dylib
-- Found PkgConfig: /nix/store/niz4jpjf9x38ja6i74mzx14vdnscf9va-pkg-config-wrapper-0.29.2/bin/pkg-config (found version &quot;0.29.2&quot;)
-- Checking for module &#x27;libraw&#x27;
--   Found libraw, version 0.21.4
-- Checking for module &#x27;libraw_r&#x27;
--   Found libraw_r, version 0.21.4
-- Found LibRaw: /nix/store/hvyf2lldrwck5k48ny56hlkmhdzfxj6z-libraw-0.21.4-lib/lib/libraw.dylib (found version &quot;0.21.4&quot;)
CMake Error at /nix/store/g1ck4iddb7q6f5qm8pa3qqss6dp3gjlm-cmake-4.1.2/share/cmake-4.1/Modules/FindPackageHandleStandardArgs.cmake:227 (message):
  Could NOT find DevIL (missing: IL_LIBRARIES ILU_LIBRARIES IL_INCLUDE_DIR)
Call Stack (most recent call first):
  /nix/store/g1ck4iddb7q6f5qm8pa3qqss6dp3gjlm-cmake-4.1.2/share/cmake-4.1/Modules/FindPackageHandleStandardArgs.cmake:591 (_FPHSA_FAILURE_MESSAGE)
  /nix/store/g1ck4iddb7q6f5qm8pa3qqss6dp3gjlm-cmake-4.1.2/share/cmake-4.1/Modules/FindDevIL.cmake:148 (find_package_handle_standard_args)
  CMakeLists.txt:278 (find_package)


-- Configuring incomplete, errors occurred!</pre>
</details>
<details>
<summary>switchfin</summary>
<pre>       |                                              ~~~~~~~~~~~~^~~~~
       |                                              operator&quot;&quot;_json
/nix/var/nix/builds/nix-build-switchfin-0.7.6.drv-0/b/source/library/borealis/library/include/borealis/extern/nlohmann/json.hpp:24387:58: warning: identifier &#x27;_json_pointer&#x27; preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
 24387 |     using nlohmann::literals::json_literals::operator &quot;&quot; _json_pointer; //NOLINT(misc-unused-using-decls,google-global-names-in-headers)
       |                                              ~~~~~~~~~~~~^~~~~~~~~~~~~
       |                                              operator&quot;&quot;_json_pointer
6 warnings generated.
6 warnings generated.
6 warnings generated.
6 warnings generated.
6 warnings generated.
6 warnings generated.
6 warnings generated.
6 warnings generated.
[100%] Linking CXX executable Switchfin.app/Contents/MacOS/Switchfin
ld: library not found for -lstdc++fs
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [CMakeFiles/Switchfin.dir/build.make:1171: Switchfin.app/Contents/MacOS/Switchfin] Error 1
make[1]: *** [CMakeFiles/Makefile2:238: CMakeFiles/Switchfin.dir/all] Error 2
make: *** [Makefile:136: all] Error 2</pre>
</details>
<details>
<summary>ytcc</summary>
<pre>E            +  where 1 = &lt;Result SystemExit(1)&gt;.exit_code

tests/test_cli.py:226: AssertionError
________________________________ test_pipe_mark ________________________________

cli_runner = &lt;function cli_runner.&lt;locals&gt;.context at 0x112c74040&gt;

    def test_pipe_mark(cli_runner):
        with cli_runner() as runner:
            result = runner(&quot;ls&quot;, subscribe=True, update=True)
            result = runner(&quot;mark&quot;, input=result.stdout)
&gt;           assert result.exit_code == 0
E           assert 1 == 0
E            +  where 1 = &lt;Result SystemExit(1)&gt;.exit_code

tests/test_cli.py:272: AssertionError
=========================== short test summary info ============================
FAILED tests/test_cli.py::test_cleanup - assert 1 == 0
FAILED tests/test_cli.py::test_pipe_mark - assert 1 == 0
================= 2 failed, 19 passed, 12 deselected in 0.89s ==================</pre>
</details>
<details>
<summary>ytui-music</summary>
<pre>219 |                     playlist_state = &amp;mut (&amp;mut (*state_ptr)).playlistbar.1;
    |                                           +++++             +

error: implicit autoref creates a reference to the dereference of a raw pointer
   --&gt; front-end/src/ui/mod.rs:220:41
    |
220 |                     artist_state = &amp;mut (*state_ptr).artistbar.1;
    |                                         ^^---------^^^^^^^^^^^
    |                                           |
    |                                           this raw pointer has type `*mut std::sync::MutexGuard&lt;&#x27;_, State&lt;&#x27;_&gt;&gt;`
    |
    = note: creating a reference requires the pointer target to be valid and imposes aliasing requirements
    = note: references are created through calls to explicit `Deref(Mut)::deref(_mut)` implementations
help: try using a raw pointer method instead; or if this reference is intentional, make it explicit
    |
220 |                     artist_state = &amp;mut (&amp;mut (*state_ptr)).artistbar.1;
    |                                         +++++             +

warning: `ytui_music` (bin &quot;ytui_music&quot;) generated 3 warnings
error: could not compile `ytui_music` (bin &quot;ytui_music&quot;) due to 3 previous errors; 3 warnings emitted</pre>
</details>
</details>

--- a/pkgs/development/python-modules/curl-cffi/default.nix
+++ b/pkgs/development/python-modules/curl-cffi/default.nix
@@ -4,6 +4,7 @@
   buildPythonPackage,
   fetchFromGitHub,
   setuptools,
+  addBinToPathHook,
   curl-impersonate-chrome,
   cffi,
   certifi,
@@ -24,14 +25,14 @@
 
 buildPythonPackage rec {
   pname = "curl-cffi";
-  version = "0.12.0";
+  version = "0.14.0b2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "lexiforest";
     repo = "curl_cffi";
     tag = "v${version}";
-    hash = "sha256-VE/b1Cs/wpZlu7lOURT/QfP7DuNudD441zG603LT4LM=";
+    hash = "sha256-JXfqZTf26kl2P0OMAw/aTdjQaGtdyTpNnhRPlwMiZNw=";
   };
 
   patches = [ ./use-system-libs.patch ];
@@ -47,13 +48,10 @@ buildPythonPackage rec {
     certifi
   ];
 
-  env = lib.optionalAttrs stdenv.cc.isGNU {
-    NIX_CFLAGS_COMPILE = "-Wno-error=incompatible-pointer-types";
-  };
-
   pythonImportsCheck = [ "curl_cffi" ];
 
   nativeCheckInputs = [
+    addBinToPathHook
     charset-normalizer
     cryptography
     fastapi
@@ -81,6 +79,8 @@ buildPythonPackage rec {
   disabledTestPaths = [
     # test accesses network
     "tests/unittest/test_smoke.py::test_async"
+    # Runs out of memory while testing
+    "tests/unittest/test_websockets.py"
   ];
 
   disabledTests = [
@@ -97,11 +97,11 @@ buildPythonPackage rec {
 
   __darwinAllowLocalNetworking = true;
 
-  meta = with lib; {
+  meta = {
     changelog = "https://github.com/lexiforest/curl_cffi/releases/tag/${src.tag}";
     description = "Python binding for curl-impersonate via cffi";
     homepage = "https://curl-cffi.readthedocs.io";
-    license = licenses.mit;
-    maintainers = with maintainers; [ chuangzhu ];
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ chuangzhu ];
   };
 }

--- a/pkgs/development/python-modules/curl-cffi/use-system-libs.patch
+++ b/pkgs/development/python-modules/curl-cffi/use-system-libs.patch
@@ -1,8 +1,6 @@
-diff --git a/scripts/build.py b/scripts/build.py
-index b705a0d..9bfcaab 100644
 --- a/scripts/build.py
 +++ b/scripts/build.py
-@@ -105,7 +105,6 @@ def get_curl_libraries():
+@@ -141,7 +141,6 @@ def get_curl_libraries():
  ffibuilder = FFI()
  system = platform.system()
  root_dir = Path(__file__).parent.parent
@@ -10,7 +8,7 @@ index b705a0d..9bfcaab 100644
  
  
  ffibuilder.set_source(
-@@ -114,9 +113,7 @@ ffibuilder.set_source(
+@@ -150,9 +149,7 @@ ffibuilder.set_source(
          #include "shim.h"
      """,
      # FIXME from `curl-impersonate`

